src/app/(auth)/signup/components/SignUpChildren.js

```diff
@@ -109,9 +116,9 @@ export default function SignUpChildren({ goToNextStep }) {
               </FormikSelect>
 
               <Button
-                onClick={async () => {
-                  await formik.submitForm();
-                  if (formik.isValid) formik.resetForm();
+                onClick={() => {
+                  setStayOnPage(true);
```

Comment:
I still do not understand why formik does not perform validation prior to submission.
[Docs](https://formik.org/docs/guides/form-submission) says that it should

Reply:
Do you want to call the handleSubmit function in onClick? This can lead to a problem: now when we call formik.submit Form(); handleSubmit doesn't work, so we'll get an error from the server or have to call manual validation